src/lib/store/index.ts

```diff
@@ -46,6 +47,7 @@ export interface ICKBAddressItem {
   notes?: string
 }
 
+// eslint-disable-next-line lingui/no-unlocalized-strings
```

Comment:
Can this eslint rule just be disabled?

Reply:
The rule is added explicitly. So we won't miss a translation by accident.